packages/synapse-react-client/src/components/Webhook/CreateWebhookModal.tsx

```diff
@@ -0,0 +1,380 @@
+import DeleteTwoToneIcon from '@mui/icons-material/DeleteTwoTone'
+import SearchOutlined from '@mui/icons-material/SearchOutlined'
+import {
+  Checkbox,
+  FormControlLabel,
+  FormGroup,
+  IconButton,
+  InputLabel,
+  TextField,
+  Switch,
+  Unstable_Grid2 as Grid,
+  Alert,
+  Box,
+  Typography,
+  Button,
+  Link,
+} from '@mui/material'
+import {
+  ErrorResponseCode,
+  SynapseEventType,
+  SynapseObjectType,
+  Webhook,
+  WebhookVerificationStatus,
+} from '@sage-bionetworks/synapse-client'
+import { upperFirst } from 'lodash-es'
+import React, { useEffect, useState } from 'react'
+import {
+  useCreateWebhook,
+  useDeleteWebhook,
+  useUpdateWebhook,
+} from '../../synapse-queries/webhook/useWebhook'
+import { ConfirmationDialog } from '../ConfirmationDialog/ConfirmationDialog'
+import { EntityFinderModal } from '../EntityFinder/EntityFinderModal'
+import { FinderScope } from '../EntityFinder/tree/EntityTree'
+import { VersionSelectionType } from '../EntityFinder/VersionSelectionType'
+import { HelpPopover } from '../HelpPopover/HelpPopover'
+import { displayToast } from '../ToastMessage/ToastMessage'
+
+type CreateWebhookModalProps = {
+  initialData?: Webhook
+  open: boolean
+  onClose: () => void
+}
+
+const NEW_ENDPOINT_DOMAIN_REQUEST_TITLE = encodeURIComponent(
+  'Request to add new domain for webhook verification',
+)
+const NEW_ENDPOINT_DOMAIN_REQUEST_SUMMARY = encodeURIComponent(
+  '<Please provide the domain(s) you wish to use and any additional details related to your request.>',
+)
+const NEW_ENDPOINT_CONTACT_FORM_URL = `https://sagebionetworks.jira.com/servicedesk/customer/portal/9/group/16/create/85?summary=${NEW_ENDPOINT_DOMAIN_REQUEST_TITLE}&description=${NEW_ENDPOINT_DOMAIN_REQUEST_SUMMARY}`
+
+function UpdateWarningWrapper(props: React.PropsWithChildren) {
```

Comment:
When the webhook is verified, add this wrapper to the endpoint field: 
![image](https://github.com/user-attachments/assets/6d9f7e31-f186-4b82-be11-981bf0d61ffb)